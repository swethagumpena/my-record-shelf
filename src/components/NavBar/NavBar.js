import React from 'react';
import styles from './NavBar.module.scss';

const NavBar = () => (
  <div className={styles.navBar}>
    <p>
      My
      {' '}
      <span className={styles.bold}>Record</span>
      {' '}
      Shelf
    </p>
  </div>
);

export default NavBar;
