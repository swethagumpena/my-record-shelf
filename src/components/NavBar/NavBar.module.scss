@import "../../styles/styles.scss";

.navBar{
    background-color: $dark-background-color;
}

.navBar p{
    font-size: 2rem;
    margin: 0;
    font-family: 'Poppins', sans-serif;
    color: white;
    padding: 1rem;
}

.bold{
    font-weight: 1000;
}