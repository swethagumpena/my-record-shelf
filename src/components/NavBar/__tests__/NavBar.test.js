import { render, screen } from '@testing-library/react';
import React from 'react';
import NavBar from '../NavBar';

describe(NavBar.name, () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  test('should match snapshot', () => {
    const { container } = render(<NavBar />);
    expect(container).toMatchSnapshot();
  });
});
