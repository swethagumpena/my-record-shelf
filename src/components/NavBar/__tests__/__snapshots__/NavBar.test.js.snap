// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<div>
  <div
    class="navBar"
  >
    <p>
      My 
      <span
        class="bold"
      >
        Record
      </span>
       Shelf
    </p>
  </div>
</div>
`;
